versions/3.0.md

```diff
@@ -1068,8 +1073,8 @@ Each content type object provides schema and examples for a the media type ident
 Field Name | Type | Description
 ---|:---:|---
 <a name="contentTypeSchema"></a>schema | [Schema Object](#schemaObject) | The schema defining the type used for the request body.
-<a name="contentTypeExamples"></a>examples | [Examples Array](#examplesArray) | Examples of the content type.
-<a name="contentTypeExample"></a>example | [Example Object](#exampleObject) | Example of the content type.
+<a name="contentTypeExamples"></a>examples | [Examples Array](#examplesArray) | Examples of the content type.  Each example in the Examples Array must be in the correct format as specified in the _content_ type.  The `examples` object is mutually exclusive to the `example` object.  Furthermore, if referencing a `schema` which contains an example, the `examples` value will _override_ the example provided by the schema.
+<a name="contentTypeExample"></a>example | [Example Object](#exampleObject) | Example of the content type.  The example object _must_ be in the correct format as specified in the _content_ type.  The `example` object is mutually exclusive to the `examples` object.  Furthermore, if referencing a `schema` which contains an example, the `example` value will _override_ the the example provided by the schema.
```

Comment:
I thought on the call Ron suggested softening the MUST to a SHOULD. (in the above para, use MUST not must as per rfc)